File: backend/utils/telegramNotifier.js
const axios = require("axios");

async function sendApplicationToTelegram(application, photoUrls = []) {
  const TELEGRAM_BOT_TOKEN = process.env.TELEGRAM_BOT_TOKEN;
  const CHAT_ID = process.env.TELEGRAM_CHAT_ID;

  let message = `📥 *New Application Received*\n\n` +
    `👤 *Name:* ${application.name}\n` +
    `🎂 *Age:* ${application.age}\n` +
    `📧 *Email:* ${application.email}\n` +
    `📱 *Phone:* ${application.contact}\n` +
    `🌍 *Nationality:* ${application.country}\n`;

  if (application.instagram) message += `📸 *Instagram:* ${application.instagram}\n`;
  if (application.tiktok) message += `🎵 *TikTok:* ${application.tiktok}\n`;
  if (application.telegram) message += `📬 *Telegram:* @${application.telegram}\n`;

  if (photoUrls.length > 0) {
    message += `\n🖼️ *Photos:*\n`;
    photoUrls.forEach((url, i) => {
      message += `🔗 [Photo ${i + 1}](${url})\n`;
    });
  }

  await axios.post(`https://api.telegram.org/bot${TELEGRAM_BOT_TOKEN}/sendMessage`, {
    chat_id: CHAT_ID,
    text: message,
    parse_mode: "Markdown"
  });
}

module.exports = { sendApplicationToTelegram };

File: backend/utils/telegram_notifier.py
import os
import requests

def send_application_to_telegram(data, photo_urls=[]):
    token = os.getenv("TELEGRAM_BOT_TOKEN")
    chat_id = os.getenv("TELEGRAM_CHAT_ID")

    message = f"📥 *New Application Received*\n\n" \
              f"👤 *Name:* {data.get('name')}\n" \
              f"🎂 *Age:* {data.get('age')}\n" \
              f"📧 *Email:* {data.get('email')}\n" \
              f"📱 *Phone:* {data.get('contact')}\n" \
              f"🌍 *Nationality:* {data.get('country')}\n"

    if data.get('instagram'):
        message += f"📸 *Instagram:* {data.get('instagram')}\n"
    if data.get('tiktok'):
        message += f"🎵 *TikTok:* {data.get('tiktok')}\n"
    if data.get('telegram'):
        message += f"📬 *Telegram:* @{data.get('telegram')}\n"

    if photo_urls:
        message += "\n🖼️ *Photos:*\n"
        for i, url in enumerate(photo_urls):
            message += f"🔗 [Photo {i+1}]({url})\n"

    requests.post(
        f"https://api.telegram.org/bot{token}/sendMessage",
        json={"chat_id": chat_id, "text": message, "parse_mode": "Markdown"}
    )
